scripts/build/BuildCompiler.js

```diff
@@ -119,4 +115,14 @@ module.exports = class BuildCompiler {
       console.log('');
     });
   }
+
+  async run () {
+    const stats = await pify(this.compiler.run.bind(this.compiler))();
```

Comment:
the bind is needed because we're passing a function to pify, and then pify is going to execute it.  when pify executes it, it won't have its original context, which is needed.  i could put this in the constructor, but i don't think that's necessary - i don't think mutating the compiler properties so that they play nice with pify is useful. the alternative would be to remove the use of pify (which we should probably do, so that the webpack documentation is easier to follow when going through these files)

Reply:
Let's discuss webpack direction before we dive deeper into this.

Reply:
as discussed, i am no longer using pify